src/vs/editor/browser/config/domFontInfo.ts

```diff
@@ -25,3 +25,14 @@ export function applyFontInfo(domNode: FastDomNode<HTMLElement> | HTMLElement, f
 		domNode.style.letterSpacing = fontInfo.letterSpacing + 'px';
 	}
 }
+
+export function applyFontInfoToStyleSheet(styleElement: HTMLStyleElement, cssClass: string, fontInfo: BareFontInfo): void {
+	styleElement.textContent = `.${cssClass} {
```

Comment:
Can you quickly check that the font info values are sanitized before we use them in the css here? This didn't matter for `applyFontInfo` since we were setting the individual style properties but I'm worried about this cases since we are constructing stylesheet source

Reply:
Good catch. They are not sanitized and they shouldn't be used this way.

Reply:
Would you be amenable to my original approach that added a `codeBlockFontSize` property to `IMarkdownRendererOptions`?